subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -9,6 +9,14 @@
 #include <net/fota_download.h>
 #include <net/download_client.h>
 #include <dfu/dfu_target.h>
+#include <pm_config.h>
+
+#ifdef PM_S1_ADDRESS
```

Comment:
This is not the only place we've done this, but we should find some other way of signifying that MCUboot is enabled.

This is the app, so in this case `#ifdef CONFIG_BOOTLOADER_MCUBOOT` should work.

Reply:
This should only be executed when MCUBoot and B0 is enabled. Only checking for `CONFIG_BOOTLOADER_MCUBOOT` is not sufficient as you might have enabled MCUBoot without having B0, and hence upgradable mcuboot.

Reply:
Ok, I see. `#if defined(CONFIG_BOOTLOADER_MCUBOOT) && defined(CONFIG_SECURE_BOOT)`?